apps/desktop/src/lib/components/EditMode.svelte

```diff
@@ -54,6 +59,15 @@
 		});
 	});
 
+	$effect(() => {
+		if (commit) {
+			authorImgUrl =
+				commit.author.email?.toLowerCase() === $user?.email?.toLowerCase()
+					? $user?.picture
+					: commit.author.gravatarUrl;
+		}
+	});
+
```

Comment:
Instead of setting this from inside an effect, let's create a derived value, please:
```ts
const authorImgUrl = $derived.by(() => {
    if (commit) {
	return commit.author.email?.toLowerCase() === $user?.email?.toLowerCase()
		? $user?.picture
		: commit.author.gravatarUrl;
    }
    return undefined;
});
```
We want to avoid setting state from inside effects if possible :)